tools/bismark/bismark_bowtie2_wrapper.xml

```diff
@@ -610,6 +610,32 @@
                 <has_text text="--pbat" />
             </assert_command>
         </test>
+
+        <test>
+            <param name="genomeSource" value="history"/>
+            <param name="own_file" value="mm10.tiny.fa.gz" />
+            <param name="sPaired" value="single"/>
+            <param name="input_singles" value="input1.fq.gzip" ftype="fastqsanger.gz"/>
+            <param name="sort_bam" value="true"/>
+            <param name="settingsType" value="custom"/>
+            <param name="suppressed_read_file" value="true"/>
+            <param name="unmapped_read_file" value="true"/>
+            <param name="bismark_stdout" value="true"/>
+            <param name="isReportOutput" value="true"/>
+
+            <output name="output_stdout" file="summary.txt" ftype="txt" lines_diff="94">
+                 <assert_contents>
+                     <has_text text="Sequences analysed in total:" />
+                     <has_text text="44115" />
+                     <has_text text="Mapping efficiency:" />
+                     <has_text text="1.3%" />
+                     <has_text text="Bismark run complete" />
+                 </assert_contents>
+            </output>
+            <output name="report_file" file="mapping_report.txt" ftype="txt" lines_diff="6"/>
+            <output name="output" file="mapped_reads_sorted.bam" ftype="bam" lines_diff="14"/>
```

Comment:
@lparsons Galaxy has by now special datatypes for special sorted BAMs, do you want to use them?

Reply:
I didn't realize that. Happy to have them used, this was simply a fix for the sort issue for one of my users (plus minimum required flake8 formatting).